### PR TITLE
docs: add apisix-variable Chinese doc & update directory

### PR DESCRIPTION
**Pull request by hf400159**
### Description

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/apisix-variable.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+APISIX 除了支持 [NGINX 变量](http://nginx.org/en/docs/varindex.html)，自身也提供了一些变量。
```
Comment (SylviaBABY) [inline docs/zh/latest/apisix-variable.md:31]:
```suggestion
APISIX 除了支持 [NGINX 变量](http://nginx.org/en/docs/varindex.html)外，自身也提供了一些变量。
```

File: docs/zh/latest/apisix-variable.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+| graphql_root_fields | core       | GraphQL 最高级别的字段。                                                          | ["hero"]          |
+| mqtt_client_id      | mqtt-proxy | MQTT 协议中的客户端 ID。                                                          |                   |
+| route_id            | core       | APISIX 路由的 ID。                                                                |                   |
+| route_name          | core       | APISIX 路由的名称。                                                               |                   |
+| service_id          | core       | APISIX 服务的 ID。                                                                |                   |
+| service_name        | core       | APISIX 服务的名称。                                                               |                   |
+| redis_cmd_line      | Redis      | Redis 命令的内容。                                                                |                   |
+| rpc_time            | xRPC       | 在 RPC 请求级别所花费的时间。                                                      |                   |
+
+你也可以创建你自己的[变量](./plugin-develop.md#register-custom-variable)。
```
Comment (SylviaBABY) [inline docs/zh/latest/apisix-variable.md:51]:
```suggestion
当然，除上述变量外，你也可以创建自定义[变量](./plugin-develop.md#register-custom-variable)。
```

File: docs/en/latest/apisix-variable.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 ---
 title: APISIX variable
+keywords:
+ - Apache APISIX
+ - API Gateway
+ - APISIX variable
+description: This is article descriptions Apache APISIX variable.
```
Comment (SylviaBABY) [inline docs/en/latest/apisix-variable.md:7]:
```suggestion
description: This article describes the variables supported by Apache APISIX.
```

File: docs/en/latest/apisix-variable.md [hunk 4]
```diff
@@ -21,25 +26,27 @@ title: APISIX variable
 #
 -->
 
+## Description
+
 Besides [Nginx variable](http://nginx.org/en/docs/varindex.html), APISIX also provides
```
Comment (SylviaBABY) [inline docs/en/latest/apisix-variable.md:31]:
```suggestion
Besides [NGINX variable](http://nginx.org/en/docs/varindex.html), APISIX also provides
```

File: docs/en/latest/apisix-variable.md [hunk 5]
```diff
@@ -21,25 +26,27 @@ title: APISIX variable
...
+| graphql_root_fields | core       | the top level fields of GraphQL.                                                    | ["hero"]       |
+| mqtt_client_id      | mqtt-proxy | the client id in MQTT protocol.                                                     |                |
+| route_id            | core       | id of Route.                                                                        |                |
+| route_name          | core       | name of Route.                                                                      |                |
+| service_id          | core       | id of Service.                                                                      |                |
+| service_name        | core       | name of Service.                                                                    |                |
+| redis_cmd_line      | Redis      | the content of Redis command.                                                       |                |
+| rpc_time            | xRPC       | time spent at the rpc request level.                                                |                |
```
Comment (SylviaBABY) [inline docs/en/latest/apisix-variable.md:50]: These descriptions need to use capital letters at the beginning.

File: docs/en/latest/config.json [hunk 6]
```diff
@@ -200,6 +200,31 @@
...
+          "type": "doc",
+          "id": "plugin-develop"
+        },
+        {
+          "type": "doc",
+          "id": "wasm"
+        },
+        {
+          "type": "link",
+          "label": "CODE_STYLE",
```
Comment (spacewander) [inline docs/en/latest/config.json:217]: What's the order of the sections?

Comment (hf400159) [reply docs/en/latest/config.json:213]: Hi, I readjusted the sorting. Here, users need to understand the content related to APISIX development and then develop it at the end. Do you have a better suggestion? @spacewander

Comment (spacewander) [reply docs/en/latest/config.json:213]: It's OK. I am just being curious.

**Review by spacewander — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Pull request merged**